test/parallel/test-cluster-worker-wait-server-close-schednone.js

```diff
@@ -0,0 +1,79 @@
+var common = require('../common');
+var assert = require('assert');
+var cluster = require('cluster');
+var net = require('net');
+
+if (cluster.isWorker) {
+
```

Comment:
There seems to be a lot of extra blank lines in the tests. Could you do away with some of them.